litmus-portal/frontend/src/components/DateTime/CustomDate/index.tsx

```diff
@@ -8,18 +8,20 @@ import useStyles from './styles';
 
 interface CustomDateProps {
   disabled: boolean;
+  selectedDate: Date | null;
+  handleDateChange: (
+    date: Date | null,
+    value?: string | null | undefined
```

Comment:
It can either be string | null right! How will value be undefined?

Reply:
@S-ayanide The default onChange event of KeyBoardDatePicker has value as string | null | undefined, 
`BaseKeyboardPickerProps.onChange: (date: Date | null, value?: string | null | undefined) => void`

Reply:
Okay so this is a KeyBoardPicker, got it